introduction.md

```diff
@@ -1,3 +1,13 @@
-I am a developer who is trying to finish the codeschool mastering github course
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?